# mode selection
uint16 mode


# ids
uint16[] id

uint16[] Goal_position
uint16[] Moving_speed
uint16[] Torque_limit



# modes

uint16 ROSA_DX_TORQUE_ON =1
uint16 ROSA_DX_TORQUE_LIMIT =2

uint16 ROSA_DX_READ =3
uint16 ROSA_DX_GOAL_POSITION=4
uint16 ROSA_DX_GOAL_SPEED =5

uint16 ROSA_DX_CHANGE_MODE =6
uint16 ROSA_DX_CHANGE_OFFSET =7

uint16 ROSA_DX_PART_UP_RIGHT = 8
uint16 ROSA_DX_PART_DOWN_LEFT = 9
uint16 DX_KINEMATICS_MOVE =10





